configs/silverstripe.json

```diff
@@ -19,7 +20,8 @@
   "nb_hits": 21730,
   "custom_settings": {
     "attributesForFaceting": [
-      "version"
+      "version",
+      "context"
```

Comment:
We could not find the following meta in all pages of your website.
```html
<meta data-react-helmet="true" name="docsearch:context" content="TBD">
```

Could you make sure that this meta is available on every page please?

Reply:
Sorry, where did you see this `TBD` tag? Could you provide a link to a page that does not have it? Should be a value of `user` or `docs` depending on the domain.

Again, probably irrelevant if we're doing a new application.

Reply:
TBD here stands for To Be Defined. Since I couldn't find the meta I have created this value. Sorry if it was not clear!

This is indeed irrelevant and it does not work like this. Please refer [to the dedicated documentation](https://community.algolia.com/docsearch/config-file.html) before submitting a PR. Let us know if some part of the documentation are not clear, we will be happy to correct it.